src/styles/setings/header.scss

```diff
@@ -0,0 +1,41 @@
+$head-marg: 50px;
+$hover-color: #00acdc;
+$text-color: #060b35;
+
+.nav {
```

Comment:
Consider adding a more descriptive class name for `.nav` to avoid potential conflicts with other navigation elements in larger projects.